cmd/tempo/app/config.go

```diff
@@ -116,9 +116,9 @@ func (c *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet) {
 
 	// Everything else
 	flagext.DefaultValues(&c.IngesterClient)
-	c.IngesterClient.GRPCClientConfig.GRPCCompression = "snappy"
+	c.IngesterClient.GRPCClientConfig.GRPCCompression = ""
```

Comment:
nit: since this is changing the default values, let's update `docs/sources/tempo/configuration/manifest.md` as well.

Reply:
I'm writing a script to generate it automatically and to the check it in CI. I will add it in another PR.